### ISSUE TITLE
Update stable version in repo from 1.1.4 to 1.1.5

### ISSUE THREAD

**Issue by GermanBluefox** [iobroker-community-adapters/ioBroker.plex#382]
# Think about update stable version to 1.1.5
**Version**: stable=**1.1.4** (416 days old) => latest=**1.1.5** (15 days old)
**Installs**: stable=**653** (73.7%), latest=**49** (5.53%), total=**886**

Click to use [developer portal](https://www.iobroker.dev/adapter/plex/releases)
Click to [edit](https://github.com/ioBroker/ioBroker.repositories/edit/master/sources-dist-stable.json#L2058)

**Do not close this issue manually as a new issue will be created if condition for update still exists.**

Please drop a comment if any reason exists which blocks updating to version 1.1.5 at this time.

Note: This is an automatically generated message and not personally authored by bluefox!
      @mcm1957 for evidence

**Comment by mcm1957** [comment]
@ticaki 
Spricht was gegen eine Aktualisiserung im STABLE repo ?

**Comment by ticaki** [comment]
@mcm1957 
Nein, außer das anscheinend der js-controller als Abhängigkeit fehlt. Wenn das trotzdem ok ist, mache ich ne manuell PR (wegen copyright year)

**Comment by mcm1957** [comment]
Ja das ist schon OK - halt für die nächste Release bitte mal fixen.

**Comment by mcm1957** [comment]
Kannst auch Copyright in README und LICENSE anpassen - OHNE neue Release zu erstellen. Reciht für Check auch dass der webbased geht.

**Comment by ticaki** [comment]
Er scheitert auch am js-controller

**Comment by GermanBluefox** [comment]
This issue seems to be outdated.

This issue suggests to update the stable version of this adapter to 1.1.5 but the current stable version is already 1.1.5.

So this issue should be closed.

@mcm1957 for evidence
